package.json

```diff
@@ -1,4 +1,36 @@
 {
+  "name": "reffy",
+  "version": "0.4.0",
```

Comment:
I'd go with 1.0 - it's been in use for a while now

also, I think we should commit to use semver - not sure if that's something that is exposed in package.json

Reply:
Following the semver format is required in package.json, but I don't think there's any way to assert in that file that we commit to follow semver conventions. I'll adjust the version number, merge and publish.